kr20.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np 
+import json
+
+from flask import Flask, escape, request
+from flask_cors import CORS
+
+app = Flask(__name__)
+CORS(app)
+
+
+@app.route('/')
+def welcome():
+    return {'message': 'Hello'}
+
+
+@app.route('/kr20/<json_array>', methods=['POST', 'GET'])
+def compute_kr20(json_array):
+    req = request.get_json(json_array)
+    return calculate_kr20(req)
+
+
+def calculate_kr20(param):
+    student_list = list(param['students'])
+    numStudents = len(student_list)
+    numQ = len(student_list[0]['itemresponses'])
+    for k in range(0, numStudents):
+        if (numQ != len(student_list[k])):
+            return {'Error: All student\'s item count must be the same'}
```

Comment:
return error in json also like {'Error': 'All student\'s item count must be the same'}